drivers/shared/executor/executor_linux.go

```diff
@@ -45,26 +45,8 @@ var (
 
 	// ExecutorCgroupMeasuredCpuStats is the list of CPU stats captures by the executor
 	ExecutorCgroupMeasuredCpuStats = []string{"System Mode", "User Mode", "Throttled Periods", "Throttled Time", "Percent"}
-
-	// allCaps is all linux capabilities which is used to configure libcontainer
-	allCaps []string
 )
 
-// initialize the allCaps var with all capabilities available on the system
-func init() {
```

Comment:
I chose to avoid using an initializer here.  It's only used once at launching executor container (and a executor process only launches a single container now), and we don't need to initialize it in main nomad/logmon/etc process.